tools/wasm_demo/one_line_demo.html

```diff
@@ -0,0 +1,20 @@
+<html>
+
+<head>
+  <link rel="icon" type="image/x-icon" href="favicon.ico" />
+  <script src="service_worker.js">
+/*
+ * Just load this script, et viola! It will install ServiceWorker to
```

Comment:
```suggestion
 * Just load this script, et voila! It will install ServiceWorker to
```
or with an accent: voilà?